02LessonTwo/homework/index.html

```diff
@@ -1,12 +1,30 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title></title>
+
+  <title>queen xenomorph</title>
+
   <link rel="stylesheet" href="css/style.css">
+
 </head>
+
 <body>
 
-<h1 style="color:red; font-size:32px">Hello, world!</h1>
+  <div>
+
+    <img src="http://static.comicvine.com/uploads/original/11115/111150831/3881735-2364554376-Alien.jpg" alt="Queen Xenomorph image" title="All hail the Queen"/>
+
+    <p>Watch out!</p>
+
+    <a href="http://roosterteeth.com/" target="_blank">roosterteeth</a>
+
+  </div>
+
```

Comment:
nice use of commenting! I wanted a few more elements in this one (h1 and ul with some li), so please go back and add these in. Great job with spacing and tabbing as well!